graphdatascience/query_runner/neo4j_query_runner.py

```diff
@@ -175,12 +180,18 @@ def call_procedure(
         def run_cypher_query() -> DataFrame:
             return self.run_cypher(query, params, database, custom_error)
 
-        if logging:
+        if self._resolve_show_progress(logging):
             job_id = self._progress_logger.extract_or_create_job_id(params)
             return self._progress_logger.run_with_progress_logging(run_cypher_query, job_id, database)
         else:
             return run_cypher_query()
 
+    def _resolve_show_progress(self, show_progress: bool) -> bool:
+        if self._show_progress is False:
+            return False
+
+        return show_progress
```

Comment:
nit, could just have 
```suggestion
        return show_progress and self._show_progress
```